cmd/lifecycle/creator.go

```diff
@@ -280,3 +285,26 @@ func (c *createCmd) registryImages() []string {
 	}
 	return registryImages
 }
+
+func (c *createCmd) platformAPIVersionGreaterThan06() bool {
```

Comment:
Not related to your PR, but should we rename all of these functions to be something like:
`platformAPIVersionIsAtLeast07`?
I think it's a bit weird that we're talking about version `0.7` and the function name indicates `0.6`.
Also, someone can theoretically think that version `0.6.1` counts since is greater than `0.6`.
@natalieparellano @jabrown85

Reply:
I agreed but there are some other references in the code that mentioned **platform06** I suggest we should do a full review of those terms and renaming them with a global strategy and avoid doing the change in just one place and keep the other ones.

Reply:
Of course! This is why I wrote that it's unrelated to your PR 😊